docs/data/material/getting-started/support/support.md

```diff
@@ -1,84 +1,67 @@
 # Support
 
-<p class="description">From community help to premium critical business support, we're here to help.</p>
+<p class="description">From community guidance to critical business support, we're here to help.</p>
 
-## Community help (free)
+## Community help
 
-The community is your first stop for questions and advice about the framework. Welcome to the community!
+Welcome to the MUI community!
```

Comment:
Hmm yeah it's a weird situation, and I agree that ultimately it belongs in a companywide doc space. But since this doc currently discusses MUI X too, and given that we don't have an equivalent page for the other Core libs, I think MUI is preferable here.